chaosaws/cloudwatch/actions.py

```diff
@@ -100,10 +98,63 @@ def remove_rule_targets(rule_name: str, target_ids: List[str] = None,
     return _remove_rule_targets(rule_name, target_ids, client)
 
 
+def put_metric_data(namespace: str,
+                    metric_data: List[Dict[str, Any]],
+                    configuration: Configuration = None,
+                    secrets: Secrets = None):
+    """
+    Publish metric data points to CloudWatch
+
+    :param namespace: The metric namespace
+    :param metric_data: A list of metric data to submit
+    :param configuration: AWS authentication configuration
+    :param secrets: Additional authentication secrets
+    :return: None
+
+    example:
+        namespace='MyCustomTestMetric',
+        metric_data=[
+            {
+                'MetricName': 'MyCustomInstanceMetric',
+                'Dimensions': [
+                    {'Name': 'InstanceId', 'Value': 'i-000000000000'},
+                    {'Name': 'Instance Name', 'Value': 'Test Instance'}
+                ],
+                'Timestamp': datetime(yyyy, mm, dd),
+                'Value': 55.55,
+                'StatisticValues': {
+                    'SampleCount': 0.0,
+                    'Sum': 0.0,
+                    'Minimum': 0.0,
+                    'Maximum': 0.0
+                },
+                'Values': [0.0, 1.0, 2.0],
+                'Counts': [1.0, 1.5, 2.6],
+                'Unit': 'Percent',
+                'StorageResolution': 60
+            }
+        ]
+
+    For additional information, consult: https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/cloudwatch.html#CloudWatch.Client.put_metric_data
+    """
```

Comment:
Just after the quotes, you can do:

```python
"""  noqa: E501
```

This will remove the warning for the entire docstring.